test/signer.spec.js

```diff
@@ -355,6 +355,20 @@ test('it respects payload exp claim (if supplied), overriding the default expire
   t.end()
 })
 
+test('it supports expiresIn as a string', t => {
```

Comment:
minor: use async tests so you don't have to remember to call t.end to end the test. simply making the test function async achieves that. I appreciate you are complying with the existing convention in this codebase, but I prefer we update it to the new syntax. t.end is only useful when doing assertions in callbacks, which is a niche use case